app/posts/how-to-use-cross-government-slack.md

```diff
@@ -0,0 +1,51 @@
+---
+title: How to use cross-government Slack
+description: Tips on how to join and make the most of the cross-government Slack workspace.
+date: 2023-06-27
+author:
+  name: Steve Messer
+  url: https://github.com/stevenjmesser
+image:
+  src: 
+  alt: 
+  opengraphImage: true
+---
+
+[Cross-government Slack](https://ukgovernmentdigital.slack.com/) is a Slack workspace for anyone working in central government, with channels on a range of topics. It’s a place to connect with other people working on digital services across government.
+
+## A bit of history
+
+The cross-government Slack was created by the [Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service) (GDS) in 2015 as a way for GDS to collaborate with other public sector digital service teams.
+
+Over time, the workspace naturally grew and became a place to share knowledge on all things about digital, data and technology in the public sector.
```

Comment:
Worth mentioning that the Slack also started to be used as an official support channel?

```suggestion
Over time, the workspace naturally grew and became a place to share knowledge on all things about digital, data and technology in the public sector. It also started to be used an an official support channel by some product teams, such as the GOV.UK Design System and GOV.UK Notify.
```